Automattic/wp-calypso
# Seller Experience: Customize Woo landing page for Sell users

#### Changes proposed in this Pull Request

Display custom content for simple sellers on the WooCommerce admin landing page. A simple seller is something who has a payment-block website and hasn't upgraded yet.

#### Testing instructions

Visit a non-seller website admin and go to `/woocommerce-installation`. You should see the following content:

![image](https://user-images.githubusercontent.com/917632/157089939-dd1d7120-6236-42b7-abd9-dddd101fe5f6.png)

Then visit the `/woocommerce-installation` page for a website that used the "Start simple" seller signup flow. You should see the following content instead:
![image](https://user-images.githubusercontent.com/917632/157090028-846fadd7-5e1e-4a10-8394-68e1d2875eee.png)


Fixes #61472
